tests/gtests/test_sum.cpp

```diff
@@ -249,10 +249,13 @@ class sum_test_t : public ::testing::TestWithParam<sum_test_params> {
                 int mant_digits
                         = dnnl::impl::nstl::numeric_limits<src_data_t>::digits;
                 int want_mant_digits = 3;
+                int digits_shift = mant_digits - want_mant_digits;
+                uint_type max_val = (uint_type)-1;
+                // Move left to keep mask value in uint_type range, move left
+                // to flush all digits but `want_mant_digits`.
```

Comment:
Is the comment even needed? ;)
```suggestion
                // Move right to clear trailing mantissa bits, move left
                // to keep `want_mant_digits` leading mantissa bits.
                // Procceeding this way avoids overflow on mask 
```

Reply:
It was shifting left only before and it didn't come to me where the problem might be. So, this is kind of documenting shift to the right and why it's necessary here.